client/src/components/core/ms-text/MsInformativeText.vue

```diff
@@ -3,7 +3,6 @@
 <template>
   <div
     class="container-textinfo"
-    :class="props.theme ?? MsTheme.Basic"
   >
     <ion-icon
       :icon="getIcon()"
```

Comment:
Might as well remove the function call and put the code directly into the template.